model/account.go

```
@@ -0,0 +1,69 @@
+
+	IsDisabled bool `json:"is_disabled" gorm:"comment:是否禁用"`
+}
+
+func (u *Account) BeforeCreate(tx *gorm.DB) (err error) {
+	if u.Password != nil {
+		if err := u.HashPassword(); err != nil {
+			return err
+		}
+	}
+
+	u.BaseModel.BeforeCreate(tx)
```

golangci-lint (1.62.2)

Error return value of `u.BaseModel.BeforeCreate` is not checked

(errcheck)